charts/function-mesh-operator/README.md

```diff
@@ -0,0 +1,188 @@
+# function-mesh-operator
+
+![Version: 0.2.7](https://img.shields.io/badge/Version-0.2.7-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+
+function mesh operator Helm chart for Kubernetes
+
+**Homepage:** <https://github.com/streamnative/function-mesh>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Function Mesh | <function-mesh@streamnative.io> | https://github.com/streamnative/function-mesh |
+
+## Source Code
+
+* <https://github.com/streamnative/function-mesh>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| (Built-in) | admission-webhook | 0.2.7 |
+
+## Values
+
+| Key | Type | Default | Description |
```

Comment:
There is no content in the description column. Is there any need to keep this column?